File: blazorBookLibrary/Security/RecaptchaVerifyResponse.cs
using System.Text.Json.Serialization;

namespace blazorBookLibrary.Security;

public class RecaptchaVerifyResponse
{
    public bool Success { get; set; }
    public double Score { get; set; }
    public string Action { get; set; }
    [JsonPropertyName("error-codes")]
    public List<string>? ErrorCodes { get; set; }
}

File: blazorBookLibrary.Shared/Slices/Security/RecaptchaVerifyResponse.fs
namespace blazorBookLibrary.Shared.Security

open System.Text.Json.Serialization

[<CLIMutable>]
type RecaptchaVerifyResponse =
    { Success: bool
      Score: float
      Action: string
      [<JsonPropertyName("error-codes")>]
      ErrorCodes: System.Collections.Generic.List<string> }